adapters/ethon/test/opentelemetry/adapters/ethon/adapter_test.rb

```diff
@@ -17,18 +17,20 @@
   before do
     exporter.reset
 
-    # these are currently empty, but this will future proof the test
-    @orig_injectors = OpenTelemetry.propagation.http_injectors
-    OpenTelemetry.propagation.http_injectors = [
-      OpenTelemetry::Trace::Propagation.http_trace_context_injector
-    ]
+    # this is currently a noop but this will future proof the test
+    @orig_propagator = OpenTelemetry.propagation.http
+    propagator = OpenTelemetry::Context::Propagation::Propagator.new(
+      OpenTelemetry::Trace::Propagation::TraceContext.text_injector,
+      OpenTelemetry::Trace::Propagation::TraceContext.text_extractor
+    )
+    OpenTelemetry.propagation.http = propagator
```

Comment:
We seem to be using this pattern in a lot of tests. Can we extract it to test helpers?

Reply:
The tests are in separate gems. There is a need to share utilities between the different instrumentation packages, but we don't have a good place them yet. I could stick this in the test_helper for the API gem, and require it here, but that doesn't seem like the best option. I'm wondering if we should punt on this until we have a common place to put it.